pkg/model/dev.go

```diff
@@ -374,6 +375,14 @@ func Read(bytes []byte) (*Manifest, error) {
 		})
 	}
 
+	for _, b := range manifest.Build {
+		if b.Name != "" {
```

Comment:
I think we should return an error here. `Name` is not supported here

Reply:
This is for parsing the relPath as Context and not as name when using BuildInfo Object